lark/lark.py

```diff
@@ -540,7 +540,7 @@ def open(cls: Type[_T], grammar_filename: str, rel_to: Optional[str]=None, **opt
             return cls(f, **options)
 
     @classmethod
-    def open_from_package(cls: Type[_T], package: str, grammar_path: str, search_paths: Tuple[str, ...]=("",), **options) -> _T:
+    def open_from_package(cls: Type[_T], package: str, grammar_path: str, search_paths: List[str]=[""], **options) -> _T:
```

Comment:
Why not `Sequence[str]`?

Reply:
I can change it to Sequence. Anything else?

Reply:
No, looks good otherwise :-)